Updates/0332_creature_ai_texts_broadcast_text_id.sql

```diff
@@ -0,0 +1,154 @@
+# ################################################################################
+# TABLE creature_ai_texts
+# ################################################################################
+
+# --------------------------------------------------------------------------------
+# ## tab -104	=> "Who challenges Naias? Puny $r, you are little better than those mindless trolls I have played against each other like so many pieces on a game board!"
+# == mal [13560] "Who challenges Naias? Puny $r, you are little better than those mindless trolls I have played against each other like so many pieces on a game board!"
+
+UPDATE creature_ai_texts SET broadcast_text_id = 13560 WHERE entry = -104;
+
+# --------------------------------------------------------------------------------
+# ## tab -321	=> "Iman m t'ief fu Fus'obeah italaf"
+# HAND mal[1910] 'Have a taste of Frostmane magics.' (from AnonXS)
+
+UPDATE creature_ai_texts SET broadcast_text_id = 1910 WHERE entry = -321;
+
+# --------------------------------------------------------------------------------
+# ## tab -374	=> "[Dwarvish] Dum mos"
+# https://classic.wowhead.com/npc=2149/dark-iron-raider
+# quotes : "Time to die, <class>."
+# HAND mal[1935] 'Time to die, $C.'
+
+UPDATE creature_ai_texts SET broadcast_text_id = 1935 WHERE entry = -374;
+
+# --------------------------------------------------------------------------------
+# ## tab -376	=> "[Orcish] Ogg kaz ko zugas kil mog Tov'nokaz Kagg"
+# COMMENT : "Orc Common Text"
+# NONE FOUND
+
+# --------------------------------------------------------------------------------
+# ## tab -377	=> "[Orcish] Tago mog regas gi mog Tov'nokaz Kagg"
+# COMMENT : "Orc Common Text"
+# NONE FOUND
+
```

Comment:
Based on the translation into German
=> "[Orcish] Ogg kaz ko zugas kil mog Tov'nokaz Kagg" -376
=> "You are no match for the Blackrock Orcs!"  bid=1875

=> "[Orcish] Tago mog regas gi mog Tov'nokaz Kagg"  -377
=> "Feel the power of the Blackrock Orcs!"  bid=1874

but not 100% sure

Reply:
https://github.com/cmangos/tbc-db/commit/5f6336d63c733cfcfdba572444dc172af46d3997#diff-8344129e35190a1fe0366238d7f8994431be263a6cfe54e26376f0479744540bR31291